platform/linuxbsd/detect.py

```diff
@@ -15,47 +15,6 @@ def can_build():
     if os.name != "posix" or sys.platform == "darwin":
         return False
 
-    # Check the minimal dependencies
-    x11_error = os.system("pkg-config --version > /dev/null")
```

Comment:
You should still check that pkgconfig is available. Can probably stay here.

Reply:
Oh, I assumed Environment.ParseConfig to be more complex than it actually is, thanks!

Reply:
Readded, although I renamed the variable to remove the reference to X11.